src/HelloWorld/Controllers/HelloWorldController.cs

```diff
@@ -18,6 +19,7 @@ public class HelloWorldController : ServiceStatusController
         [HttpGet]
         public string Get()
         {
+            Logger.Info(HttpContext.Request, "info level message with no object");
```

Comment:
Use a relevant message in this log

for example

```suggestion
            Logger.Info(HttpContext.Request, "starting hello api");
```

Reply:
Changed!